src/mpi/coll/op/oplxor.c

```diff
@@ -55,10 +55,6 @@ void MPIR_LXOR(void *invec, void *inoutvec, MPI_Aint * Len, MPI_Datatype * type)
                 MPIR_OP_TYPE_GROUP(C_INTEGER_EXTRA)
                 MPIR_OP_TYPE_GROUP(FORTRAN_INTEGER)
                 MPIR_OP_TYPE_GROUP(FORTRAN_INTEGER_EXTRA)
-                /* We previously supported floating point types, although I question
-                 * their utility in logical boolean ops [goodell@ 2009-03-16] */
-                MPIR_OP_TYPE_GROUP(FLOATING_POINT)
-                MPIR_OP_TYPE_GROUP(FLOATING_POINT_EXTRA)
```

Comment:
Should remove these later in the file in `MPIR_LXOR_check_dtype`? It looks like we left floating pointing types as valid there for all the logical functions.

Reply:
fixed.

Reply:
There were also instances of these in opland.c and oplor.c. I just pushed an extra commit to the branch, which can be squashed.